Public/Sdk/Experimental/NodeJs/Yarn/yarn.dsc

```diff
@@ -86,15 +86,11 @@ export function install(args: Arguments) : Result {
 
     if (useAuthenticatedPackageFeed && Context.getCurrentHost().os === "win") {
 
-        if (Environment.hasVariable("NUGET_CREDENTIALPROVIDERS_PATH")) {
-            const nugetCredentialProviderPath = Environment.getDirectoryValue("NUGET_CREDENTIALPROVIDERS_PATH");

```

Comment:
The nuget_credentialproviders_path is still needed for the selfhost (builds done locally by IC's)

Reply:
You might have hit cache hits, could you try:

```
rmdir /s/q out\objects
rmdir /s/q %LOCALAPPDATA%l\Yarn
```
I beleive this will cause yarn to redownload the packages from our internal Azure DevOps feed which will require authentication.


---
In reply to: [310272241](https://github.com/microsoft/BuildXL/pull/695#discussion_r310272241) [](ancestors = 310272241)

Reply:
the variables below are all good to externalize for cloudbuild environment only.

---
In reply to: [310272871](https://github.com/microsoft/BuildXL/pull/695#discussion_r310272871) [](ancestors = 310272871,310272241)

Reply:
added it back